src/main/java/io/github/thebusybiscuit/slimefun4/core/networks/cargo/CargoNetworkTask.java

```diff
@@ -111,7 +111,8 @@ private void routeItems(Location inputNode, Block inputTarget, int frequency, Ma
                 if (inv.getItem(previousSlot) == null) {
                     inv.setItem(previousSlot, stack);
                 }
-                else {
+                // Making sure the item cannot be added to another slot before dropping it on ground
+                else if((stack = inv.addItem(stack).get(0)) != null){
```

Comment:
I think this is a bit messy. Maybe create a variable of inv.addItem().get() and then check if it's null. I don't quite understand how checking if the inventory is empty can duplicate the item though.

Reply:
> 
> 
> I think this is a bit messy. Maybe create a variable of inv.addItem().get() and then check if it's null. I don't quite understand how checking if the inventory is empty can duplicate the item though.

addItem is adding the item, and returns the results. If it tried to add 64 of an item, but only 28 of it could be added, it would return the item with an amount of 36. If you only check if it's empty, and you drop the original item (the 64 one), you drop 64, however 28 were already deposited. Aka, the items was duped.

Instead of checking why the PR is not good, just accept it lmao
It's such a simple additional, I don't understand why you keep refusing to accept it

Reply:
We aren't checking whether the PR is good, we are doing code reviews to ensure changes won't break anything.

There is always a bias with the person who wrote the changes so we require 2 approving reviews from maintainers to ensure nothing goes unnoticed.

Especially in such a messy area of the codebase there really isn't anything like a "simple addition".
I agree today's reviews didnt go as usual, but its still important to review changes before merging and to ask questions and get a better understanding of the situation.

Reply:
Because it's still hard to read. Can't you separate it into a couple lines. Like 1 for the variable and other checking if that variable is null. Just add spaces around the brackets and instead of assigning the variable inside if statement, assign it in an upper line.

Reply:
> 
> 
> Because it's still hard to read. Can't you separate it into a couple lines. Like 1 for the variable and other checking if that variable is null. Just add spaces around the brackets and instead of assigning the variable inside if statement, assign it in an upper line.

it makes no sense to separate the lines. If you really want to separate them, go for it.
You are only looking for reasons why to not accept a literally cleaned and working code

Reply:
> > 
> > 
> > Because it's still hard to read. Can't you separate it into a couple lines. Like 1 for the variable and other checking if that variable is null. Just add spaces around the brackets and instead of assigning the variable inside if statement, assign it in an upper line.
> 
> it makes no sense to separate the lines. If you really want to separate them, go for it.
> You are only looking for reasons why to not accept a literally cleaned and working code

And you are now looking for ways to start a fight. In nowhere of sf code, it was used. Your code does not follow the code standards of this repo. All changes we have asked for can easily be done even on github website and on mobile. You could have done them already while writing all these comments.

Reply:
No need to fight, both of you. Mistakes have been made here by multiple people. Formatting and docs can always be improved further. But no need to drag this on even longer.

Reply:
I am not starting a fight. I don't see a point in changing it, and if you really want to do that - go for it.
I don't know what are your coding rules, and I am 100% sure that if I change it you'll have complaints again.
I am not gonna continue this pointless discussion. If you want to change it, do it. If not, then fine - the code is good like this.
And as you said, "All changes we have asked for can easily be done even on github website and on mobile. You could have done them already while writing all these comments." - then just do the changes instead of wasting my time.